packages/self_storing_input/test/self_storing_text_test.dart

```diff
@@ -30,15 +31,14 @@ void main() {
 
     testWidgets('overlay renders successfully if trivial parameters provided.',
         (WidgetTester tester) async {
+      var style = SelfStoringTextStyle();
```

Comment:
Is this a test that exposes the bug?

Reply:
No, the bug is not covered. I plan to add test coverage with next change.

Reply:
It is not trivial to cover :)